### DISCUSSION TITLE
Empty response on access

### DISCUSSION THREAD

**Discussion by apfelbox** [plausible/analytics#1324] [Self-Hosted Support] [answered]
Hi,

so I am trying to self-host Plausible.
I installed plausible and run it with docker-compose. I changed the external port to `8123`:

![Capture 2021-09-13_15-19-22](https://user-images.githubusercontent.com/1032411/133092125-1c62acf0-3f27-4a40-9e83-9d06f24971ef.png)

The system is running, however if I try to access it I just get an empty response:
![Capture 2021-09-13_15-20-03](https://user-images.githubusercontent.com/1032411/133092159-4e7d9aab-83ce-4221-ae51-1b337c9b3816.png)

Is there some way to find out what the error is?

Thanks
Jannik

**Comment by apfelbox** [comment, marked as answer]
So, I finally found the issue.

I had quotes in my URL in my env file:

```env
# this is wrong (it's not actually, but according to Docker's env parser it is)
BASE_URL="https://example.org"

# this works
BASE_URL=https://example.org
```

This lead to the container continuously crashing and restarting and that led to an empty response.

**Reply by metmarkosaric** [reply]
thanks for the update @apfelbox!
